### PR TITLE
PCQ-880: Updated to add fortify pipeline

### PR DESCRIPTION
**Pull request by davejones74**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PCQ-880

### Change description ###
Fortify integration with frontend.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: test/java/config/fortify-client.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=65442
```
Comment (kaanaktas) [inline test/java/config/fortify-client.properties:1]: EOF will be good

File: test/java/.gitignore [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+.idea/
+*.iws
+*.iml
+*.ipr
+
+### Gradle
+.gradle/
+out/
+bin/
+build/
```
Comment (kaanaktas) [inline test/java/.gitignore:18]: EOF will be good

File: test/java/gradlew [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+# Collect all arguments for the java command, following the shell quoting and substitution rules
+eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$APP_ARGS"
+
+# by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
+if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then
+  cd "$(dirname "$0")"
+fi
+
+exec "$JAVACMD" "$@"
```
Comment (kaanaktas) [inline test/java/gradlew:172]: EOF will be good

File: test/java/settings.gradle [hunk 4]
```diff
@@ -0,0 +1 @@
+rootProject.name = 'pcq-frontend'
```
Comment (kaanaktas) [inline test/java/settings.gradle:1]: EOF will be good

File: test/java/config/fortify-client.properties (hunk 1 above)
Comment (davejones74) [reply test/java/config/fortify-client.properties]: Thanks, now resolved.

**Review by kaanaktas — APPROVED** [review]
LGTM

**Pull request merged**